certificates/certificate.go

```diff
@@ -1,13 +1,11 @@
 package certificates
 
-import (
-	"crypto/x509"
-)
+import ()
```

Comment:
Is this something InteliJ does? I believe goimports itself would just remove the line if the are no imports.